wbooks/scss/application.scss

```diff
@@ -0,0 +1,84 @@
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+//FILTER 
+.search {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
```

Comment:
This is the default value of `display: flex` so there's no actual need for this :smile: